src/lib/openapi/spec/search-events-schema.ts

```diff
@@ -0,0 +1,29 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const searchEventsSchema = {
+    $id: '#/components/schemas/searchEventsSchema',
+    type: 'object',
+    properties: {
+        type: {
+            type: 'string',
+        },
+        project: {
+            type: 'string',
+        },
+        feature: {
+            type: 'string',
+        },
+        query: {
+            type: 'string',
+        },
+        limit: {
+            type: 'number',
```

Comment:
1. should we set the type to "integer"? 
2. Should we set the minimum value to "1"? 
3. should we define a "maximum" value (e.g. 1000)?
4. should we define the default value (100)?

Reply:
Yeah, that's much better!